src/cmd_line.c

```diff
@@ -56,3 +60,23 @@ void cmd_line_attempt(const char *line) {
     memset(line_copy, '\0', KEYBOARD_BUFFER_SIZE);
     return;
 }
+
+void cmd_line_help(const char *args) {
```

Comment:
Just to follow the style of each terminal function. Gives us flexibility in case we want to add flag handlers for a call to "help"